page_test.go

```diff
@@ -139,6 +139,16 @@ func (s *S) TestSetViewport() {
 	s.NotEqual(int64(317), res.Get("0").Int())
 }
 
+func (s *S) TestGetViewport() {
+	page := s.browser.Page(srcFile("fixtures/scroll.html"))
+	defer page.Close()
+	page.Viewport(1200, 400, 0, false)
+	view, _ := page.GetViewportE()
+	res := page.Eval(`() => ({w: document.body.clientWidth, h: document.body.clientHeight})`)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/clientWidth

It will exclude the scrollbar size. We should use `innerWidth` and `innerHeight` here right?

Reply:
fair point, I'm tweaking the tests rn because I don't know why it failed for Linux

Reply:
Ah, that's a bug of my test, it not stable 100% yet. I will optimize it later. If some unrelated test fails you can just rerun the CI.

Reply:
Exactly that's the one.

So, now we have `window.innerWidth` and `window.innerHeight` passing tests, seems like we're on the right track!